### ISSUE TITLE
Viewfactor File missing

### ISSUE THREAD

**Issue by jt3761** [nemocrys/test-cz-induction#1]
Hi,

I tried running your code, but i get an error that view factor file is missing. Can you tell me whether elmer calculates the viewfactors and stores them in .dat file or whether we need to provide the .dat file ? 
![image](https://github.com/nemocrys/test-cz-induction/assets/91514030/dd239e35-b6c3-4dba-b66e-829ecf62f6aa)

**Comment by arvedes** [comment]
The ViewFactors program, which comes with Elmer, should generate the file you're missing. Can you provide some additional information about your setup? You are not using the Docker image, right?
I had the same problem on Windows once when Elmer was not added to the system path during installation.

**Comment by jt3761** [comment]
Yes I am not using the Docker image. I was running it using jupyter notebook. I made few changes to import the correct libraries. I also tried running the 2d heat conduction cases for a rectangle and for crystal/melt geometry but they seem to run fine.

**Comment by arvedes** [comment]
Do you use Windows or Linux? How did you install Elmer? Can you check if you can call `ViewFactors` in a terminal?
